client/src/components/Formik/Checkbox.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Field } from "formik";
+import { FormControlLabel, Switch } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  isChef: {
+    marginTop: theme.spacing(-3),
+    height: theme.spacing(4),
+    "& .MuiTypography-root.MuiFormControlLabel-label.MuiTypography-body1": {
+      fontWeight: 900,
+      fontSize: 12,
+    },
```

Comment:
If you end up wanting all of your form labels to have this styling, you could try throwing this in a [theme override ](https://material-ui.com/customization/components/#global-theme-override)

Reply:
Thanks Nellie, I will look into it and I will edit this in my next pull request. Now I have to delete this, I have the same line in the demo pull request.